src/constants.py

```diff
@@ -39,3 +39,4 @@
 CHARMED_MYSQL_XBSTREAM_LOCATION = "xbstream"
 XTRABACKUP_PLUGIN_DIR = "/usr/lib64/xtrabackup/plugin"
 MYSQLD_DEFAULTS_CONFIG_FILE = "/etc/mysql/my.cnf"
+TMP_DIR = "/tmp"
```

Comment:
Unfortunately, the full backup would be stored in the tmp directory. The only alternative would be to store the backup in the data directory (which will result in the temp backup directory to appear in the backup). i recall @delgod mentioning that we should try, but it's not a blocker to use `/tmp`

Reply:
This will backfire. The only question when. Up2Mykola here.

LGTM from my side, IMHO this should not block migration to our OCI.
Could be forked as a bugreport.

Reply:
Important note: we are not actually storing the full backup in the dir (because of the `--stream` option).
It is needed for temporary files, like buffer pool dump.
It is not the best idea to use `/tmp` - the reason: it can be not enough space.
So, imho, it is better to create a temp dir in `/var/lib/mysql/#some-tmp-dir-path`